dev.Dockerfile

```diff
@@ -1,41 +1,10 @@
-FROM golang:1.22-alpine as build
-
-ENV GOPRIVATE="github.com/DragonOps-io/*"
-RUN apk add --no-cache file git rsync openssh-client
-RUN mkdir -p -m 0700 ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts
-
-WORKDIR /app
-
-RUN --mount=type=ssh <<EOT
-  set -e
-  echo "Setting Git SSH protocol"
-  git config --global url."git@github.com:".insteadOf "https://github.com/"
-  (
-    set +e
-    ssh -T git@github.com
-    if [ ! "$?" = "1" ]; then
-      echo "No GitHub SSH key loaded exiting..."
-      exit 1
-    fi
-  )
-EOT
-
-COPY . .
-
-# Leverage the GitHub Actions cache for Go modules
-RUN --mount=type=cache,target=/go/pkg/mod \
-    --mount=type=cache,target=/root/.cache/go-build \
-    --mount=type=ssh \
-    CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o .
-
-
 FROM 851725405730.dkr.ecr.us-east-1.amazonaws.com/dev-worker:latest as worker
 
 FROM alpine:3.18.3
 
 WORKDIR /app
 RUN apk add --no-cache bash git age aws-cli wireguard-tools
-COPY --from=build /app/orchestrator .
+COPY orchestrator .
```

Comment:
How come we don't need the --from=build /app/ here?

Reply:
@polycont Before, i was building the go binary in the Dockerfile itself, and copying it from the original "base" image. Now, I am building the binary in the pipeline GitHub runner (CI server) so that we can take advantage of go module caching! :) So we aren't copying the binary from the multistage build anymore, but instead just from the pipeline directory. Yo uuse `--from` to specify a previous docker build stage